docs/library/esp.rst

```diff
@@ -62,12 +62,18 @@ Functions
 
 .. function:: flash_erase(sector_no)
 
-.. function:: osdebug(level)
+.. function:: osdebug(uart_no, [level])
```

Comment:
The changes below are not fully correct.

On esp8266 the behaviour for `osdebug(uart_no)` is:
- pass `None` to disable debug logging
- pass an integer UART number to enable logging on that UART

For esp32 it's `osdebug(uart_no, level=LOG_DEBUG)` with:
- `uart_no` is `None` to disable logging
- otherwise logging is enabled and the optional `level` specifies the level, with default LOG_LOCAL_LEVEL which I think is LOG_DEBUG (?)

Reply:
@dpgeorge This also wasn't quite right, for a simple function it has a lot of complexities!

Have pushed a new commit which I've re-tested to double check behaviour matches docs..

Reply:
Thanks!